raspberry/bzzz/sensors/gnss.py

```diff
@@ -92,18 +94,23 @@ def __get_measurements_in_background_t(self, serial_path, baud):
                 tokens = line.split(",")
                 msg_key = tokens[0]
 
+
                 if msg_key == "$GNGLL" and tokens[1] and tokens[3]:
                     lat_deg = int(float(tokens[1]) / 100)
                     lat_min = float(tokens[1]) % 100
-                    latitude = deg_min_sec_to_decimal(lat_deg, lat_min, 
+                    latitude = _deg_min_sec_to_decimal(lat_deg, lat_min, 
                                                       tokens[2])
 
                     lon_deg = int(float(tokens[3]) / 100)
                     lon_min = float(tokens[3]) % 100
-                    longitude = deg_min_sec_to_decimal(lon_deg, lon_min, 
+                    longitude = _deg_min_sec_to_decimal(lon_deg, lon_min, 
                                                        tokens[4])
-                elif msg_key == "$GPGSV" and len(tokens) > 5:
-                    altitude = float(tokens[5])
+                    
+                elif msg_key == "$GNGGA" and len(tokens) > 10:
+                    try:
+                        altitude = float(tokens[9])
+                    except:
+                        print("err")
```

Comment:
Do we ned this Print?

Reply:
That print was just for debugging purposes. it can now be removed.